services/API-service/src/api/event/event.service.ts

```diff
@@ -496,6 +583,31 @@ export class EventService {
       eapActions: [],
       alertLevel: this.getAlertLevel(area),
     }));
+
+    const highestAlertLevels = this.getHighestAlertLevelPerEvent(areas);
+    return areas.filter(
+      (area) =>
+        area.alertLevel === highestAlertLevels[area.eventName || 'unknown'],
+    );
+  }
+
+  public getHighestAlertLevelPerEvent(
+    areas: { alertLevel: AlertLevel; eventName: string }[],
```

Comment:
this change fixes the issue that highest alert level across event was applied as filter in National View